android/engine/src/main/java/org/smartregister/fhircore/engine/rulesengine/RulesFactory.kt

```diff
@@ -70,7 +70,8 @@ constructor(
         mutableMapOf<String, Any>(
           "Timber" to Timber,
           "StringUtils" to Class.forName("org.apache.commons.lang3.StringUtils"),
-          "RegExUtils" to Class.forName("org.apache.commons.lang3.RegExUtils")
+          "RegExUtils" to Class.forName("org.apache.commons.lang3.RegExUtils"),
+          "math" to Class.forName("java.lang.Math")
```

Comment:
Maybe we should maintain the casing for consistency? i.e. `Math` instead of `math`

Reply:
done